apidoc/Titanium/UI/TextField.yml

```diff
@@ -695,3 +691,24 @@ properties:
   - name: length
     summary: Number of characters selected.
     type: Number
+---    
+name: Padding
+summary: Dictionary object of parameters for the <Titanium.UI.TextField.padding>  that describes the padding
+since: "5.4.0"
+platforms: [android, iphone, ipad]
+properties:
+  - name: left
+    type: Number
+    summary: Left padding
+    
+  - name: right
+    type: Number
+    summary: Right padding
+    
+  - name: top
+    type: Number
+    summary: Top padding
+    
+  - name: bottom
+    type: Number
+    summary: Bottom padding
```

Comment:
iOS only allows left and right padding because an input should always be vertically centered. We should consider doing the same with Android, so we don't need to handle different behaviors across platforms. Also please deprecate the old properties.